### PR TITLE
Color and Anchor Link Nav Style Updates

### PR DESCRIPTION
**Pull request by shanecmiller23**
- Update UI styling for right anchor list navigation
- Update purple colors used on Home page graphics and links

### CODE REVIEW

File: themes/upbound/assets/scss/_home.scss [hunk 1]
```diff
@@ -76,7 +76,7 @@
         .about-link {
             display: inline;
             a {
-                color: #6553C0 !important;
+                color: #7D4CD1 !important;
```
Comment (julzvilla) [inline themes/upbound/assets/scss/_home.scss:79]: Nitpick: could you use the variable `$up-purple-400` here instead? just to keep things consistent and easier to update down the link

**Review by julzvilla — CHANGES_REQUESTED** [review]

**Comment by shanecmiller23** [comment]
@julzvilla made that change

**Review by julzvilla — APPROVED** [review]
Looks good!

**Comment by shanecmiller23** [comment]
@julzvilla updated tables here too now

**Pull request merged**